src/test/java/org/gridsuite/useridentity/oidcreplication/server/UserIdentityDtoServiceTest.java

```diff
@@ -0,0 +1,115 @@
+package org.gridsuite.useridentity.oidcreplication.server;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import java.util.Map;
+import java.util.stream.Stream;
+
+import org.gridsuite.useridentity.oidcreplication.server.dto.UserIdentity;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+import org.springframework.boot.test.mock.mockito.MockBean;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+
+@SpringBootTest(webEnvironment = WebEnvironment.NONE, properties = {
+    "spring.autoconfigure.exclude=" +
+          "org.springframework.boot.autoconfigure.jdbc.DataSourceAutoConfiguration," +
+          "org.springframework.boot.autoconfigure.liquibase.LiquibaseAutoConfiguration"
+})
```

Comment:
is it to improve test run duration ?
I'm not sure if I prefer removing it so we can remove this as well :
```
@MockBean
UserIdentityRepository excluded;
```

Reply:
test run time yes by a small amount because liquibase in h2 inmemory is fast, but also reduces logs, and also reduces potential problems (ie the test still works if we break the database). It's a good thing imho to use the minimal context that allows to run the test, we should do it more